publisher/proto/publisher.proto

```diff
@@ -11,11 +11,13 @@ message Request {
         optional string LogURL = 2;
         optional string LogPublicKey = 3;
         optional bool precert = 4;
+        optional bool storeSCT = 5;
```

Comment:
Why do you need a request parameter for this? Given that we aren't submitting final logs today, we can go ahead and switch the behavior for final certs unilaterally.

Reply:
This is for deployability considerations. Since embedded SCTs are still feature flagged if we changed this to never store SCTs for plain certificates and then needed to disable the embedded SCT feature the old behavior wouldn't work and SCTs would never get stored and ocsp-updater would think it had to retrieve SCTs for every issued certificate.

Reply:
Sounds reasonable!